storage/ndb/rest-server/rest-api-server/internal/feature_store/metadata.go

```diff
@@ -0,0 +1,272 @@
+/*
+ * This file is part of the RonDB REST API Server
+ * Copyright (c) 2023 Hopsworks AB
+ *
+ * This program is free software: you can redistribute it and/or modify
+ * it under the terms of the GNU General Public License as published by
+ * the Free Software Foundation, version 3.
+ *
+ * This program is distributed in the hope that it will be useful, but
+ * WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE. See the GNU
+ * General Public License for more details.
+ *
+ * You should have received a copy of the GNU General Public License
+ * along with this program. If not, see <http://www.gnu.org/licenses/>.
+ */
+
+package feature_store
+
+import (
+	"encoding/json"
+	"fmt"
+	"sort"
+	"strings"
+	"time"
+
+	"github.com/patrickmn/go-cache"
+
+	"hopsworks.ai/rdrs/internal/dal"
+	"hopsworks.ai/rdrs/internal/log"
+)
+
+type FeatureViewMetadata struct {
+	FeatureStoreName     string
+	FeatureStoreId       int
+	FeatureViewName      string
+	FeatureViewId        int
+	FeatureViewVersion   int
```

Comment:
I understand that it is more structure by introducing FeatureStore struct. But since there is no operation on FeatureStore struct, the current way is more simple.